src/mmw/mmw/settings/base.py

```diff
@@ -525,6 +525,19 @@ def get_env_setting(setting):
                 'zoom': 0
             }
         },
+        'nlcd_soil_tr55': {
```

Comment:
Decided to make a new one instead of changing the existing one because I wasn't sure if that was used elsewhere.